packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1,7 +1,20 @@
 import type { Browser, MultiRemoteBrowser } from 'webdriverio'
 import type { Capabilities } from '@wdio/types'
+import logger from '@wdio/logger'
 
-import { BROWSER_DESCRIPTION } from './constants'
+import got from 'got'
+import { hostname, platform, type, version, arch } from 'os'
+import { promisify } from 'util'
+import gitRepoInfo from 'git-repo-info'
+import gitconfig from 'gitconfiglocal'
+import { Frameworks } from '@wdio/types'
+
+import { BROWSER_DESCRIPTION, DATA_ENDPOINT } from './constants'
+import { ITestCaseHookParameter } from '@cucumber/cucumber/lib/support_code_library_builder/types'
+import { UserConfig } from './types'
```

Comment:
Let's have wdio related imports next to other wdio imports.

```suggestion
import type { UserConfig } from './types'
```